R/gateUtil.R

```diff
@@ -133,7 +133,7 @@ compoundGateCreate = function(body, names, gid, gids,
 
   body = c(body, list(
     parentPopulationId = jsonlite::unbox(parentPopulationId),
-    names = names,
+    #names = names,
```

Comment:
Looks like a pending item?

Reply:
`Outdated` also; I ended up deleting the "names" parameter here. Because each population has it's own `name`, this would quadruplicate each population.

Reply:
Ah -- I usually review commits one at a time, so I saw these two outdated things.